core/src/main/scala/org/apache/spark/internal/config/ConfigEntry.scala

```diff
@@ -264,7 +264,9 @@ private[spark] class FallbackConfigEntry[T] (
     version
   ) {
 
-  override def defaultValueString: String = s"<value of ${fallback.key}>"
+  override def defaultValue: Option[T] = fallback.defaultValue
```

Comment:
Hm, if the fallback is set, the default value here isn't correct anymore .. right

Reply:
Also this will change the docs of configurations (https://spark.apache.org/docs/latest/configuration.html#runtime-sql-configuration) and it won't document that which one is fallback anymore.

Reply:
yeah, I think we don't need to show the fallback config that might be marked as `Deprecated`.